client-java/instrumentation/pom.xml

```diff
@@ -47,6 +47,12 @@
             <artifactId>mockito-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+        </dependency>
```

Comment:
same issue as above. cannot have it here like this. let's discuss on chat

Reply:
I will replace the GSON library with Jackson.